phoenixRest/views/friend_request/__init__.py

```diff
@@ -0,0 +1,59 @@
+from pyramid.view import view_config, view_defaults
+
+from pyramid.authorization import Allow, Everyone
+
+from phoenixRest.models.core.user import User
+from phoenixRest.models.core.friendship import Friendship
+
+from phoenixRest.utils import validate
+from phoenixRest.resource import resource
+
+from phoenixRest.roles import ADMIN
+
+from sqlalchemy import and_, or_, extract, null
+
+@resource(name='friend_request')
+class FriendRequestResource(object):
+    __acl__ = [
+        (Allow, ADMIN, 'create'),
```

Comment:
No reason to allow ADMIN if everyone is allowed.

Also since you are logged in i recommend `Authenticated` instead. I make this mistake in my code